lua/user/plugins/extras/lang/python.lua

```diff
@@ -17,4 +17,25 @@ return {
       if type(opts.ensure_installed) == "table" then vim.list_extend(opts.ensure_installed, { "python" }) end
     end,
   },
+  {
+    "raimon49/requirements.txt.vim",
+    event = "VeryLazy",
+  },
+  {
+    "AckslD/swenv.nvim",
+    ft = "python",
+    event = { "VeryLazy" },
```

Comment:
this would load it after startup, is this important? if you remove that line it would get loaded on a python file